.github/workflows/slack.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     steps:
     - name: Get current star count
       run: |
-        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/layer5io/meshery' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
+        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/layer5io/service-mesh-labs' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/meshery/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/service-mesh-labs/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

Comment:
```suggestion
        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred Service Mesh Labs! (https://github.com/layer5io/service-mesh-labs/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

Reply:
Thank you, @mitruly. Looks real good. Here's a final item to consider incorporation of.